logproxy-client/src/main/java/com/oceanbase/clogproxy/client/config/ClientConf.java

```diff
@@ -16,7 +16,7 @@
 /** The class that defines the constants that are used to generate the connection. */
 public class ClientConf extends SharedConf {
     /** Client version. */
-    public static final String VERSION = "1.0.1";
+    public static String VERSION = "1.0.1";
```

Comment:
The variables whose name consisting of capital letters and underscores should be treated as constant (default) values.